internal/pkg/service/appsproxy/dataapps/api/config.go

```diff
@@ -9,11 +9,11 @@ import (
 )
 
 type AppProxyConfig struct {
-	ID             string         `json:"-"`
-	Name           string         `json:"name"`
-	UpstreamAppURL string         `json:"upstreamAppUrl"`
-	AuthProviders  []AuthProvider `json:"authProviders"`
-	AuthRules      []AuthRule     `json:"authRules"`
+	ID             string     `json:"-"`
```

Comment:
There is no reason to prefix each type with `Auth` or `Proxy`, it is all about it.